arrangor/src/main/kotlin/no/nav/amt/tiltak/arrangor/ArrangorRepository.kt

```diff
@@ -28,40 +28,34 @@ open class ArrangorRepository(
 
 
     fun upsert(
+		id: UUID,
 		navn: String,
 		organisasjonsnummer: String,
 		overordnetEnhetNavn: String?,
 		overordnetEnhetOrganisasjonsnummer: String?,
     ): ArrangorDbo {
         val savedArrangor = getByOrganisasjonsnummer(organisasjonsnummer)
 
-        if (savedArrangor != null) {
-            return savedArrangor
-        }
-
-        val sql = """
-			INSERT INTO arrangor(id, overordnet_enhet_organisasjonsnummer, overordnet_enhet_navn, organisasjonsnummer, navn)
-			VALUES (:id,
-					:overordnetEnhetOrganisasjonsnummer,
-					:overordnetEnhetNavn,
-					:organisasjonsnummer,
-					:navn)
-		""".trimIndent()
-
-        val id = UUID.randomUUID()
-
-        val parameters = MapSqlParameterSource().addValues(
-            mapOf(
-                "id" to id,
-                "navn" to navn,
-                "organisasjonsnummer" to organisasjonsnummer,
-                "overordnetEnhetOrganisasjonsnummer" to overordnetEnhetOrganisasjonsnummer,
-                "overordnetEnhetNavn" to overordnetEnhetNavn,
-            )
-        )
-
-        template.update(sql, parameters)
-
+		if (savedArrangor == null) {
```

Comment:
Veldig bra å få bort rariteten med en upsert som aldri oppdaterer! Syns her at man også bør bruke `ON CONFLICT` for å håndtere oppdateringer